### ISSUE TITLE
QuickTile PPA / Clean Install

### ISSUE THREAD

**Issue by m4l3z** [ssokolow/quicktile#72]
Hello there, 

Would it be possible to have a PPA for QuickTile ? it would make the updating process much easier. 
I don't know much about this but if I can help in anyway hit me up. 
I'm on linux mint 18.1 ( ubuntu xenial ) (debian stretch)  

Cheers,

**Comment by ssokolow** [comment]
There isn't really anything preventing QuickTile from having a PPA. In fact, being able to split QuickTile into multiple files without worry would be a great motivator for me to work on it more.

The only real holdup is that last time I tried to learn to make DEB packages, the docs that Google turned up were horrendous (and the official Debian viewpoint seemed to be "Don't try to be your own package maintainer. You'll only screw it up."), I got demoralized and, since then, QuickTile's installation story has been Good Enough™ that I've always had things that seemed like a more productive use of my time.
